getMultimedias.js

```diff
@@ -23,13 +23,16 @@
 
       const [yearElement, runtimeElement] = div.querySelectorAll('.dli-title-metadata-item');
 
-      const year = yearElement.textContent;
-      const runtime = runtimeElement.textContent;
+      const year = yearElement ? yearElement.textContent.trim() : '';
+      const runtime = runtimeElement ? runtimeElement.textContent.trim() : '';
 
       // const genres = getTextContentIfExists(div, ".genre");
+	  const voteCountElement = div.querySelector('.ipc-rating-star--voteCount');
+      const voteCount = voteCountElement ? voteCountElement.textContent.trim() : '';
       const ratingIMDB = div.querySelector('[data-testid="ratingGroup--imdb-rating"]').textContent.replace(/\u00A0/g, ' ').split(' ')[0];
       const ratingMetac = getTextContentIfExists(div, ".metacritic-score-box");
-
+	  
+      const story = div.querySelector('.ipc-html-content-inner-div').textContent;
```

Comment:
Could we rename all `story` stuff to `plot` please?

Reply:
ok